sdk/c-examples/README.txt

```diff
@@ -0,0 +1,7 @@
+Currently no C applications are built automatically as we have chosen
```

Comment:
Can this be converted to a Markdown file?  We've no other `.txt` files in the repository.

Reply:
Done.